code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm

```diff
@@ -47,7 +47,7 @@
 
 /datum/eldritch_knowledge/cold_snap
 	name = "Aristocrat's Way"
-	desc = "Makes you immune to cold temperatures, you can still take damage from lack of pressure."
+	desc = "Makes you immune to cold temperatures, and you no longer need to breathe, you can still take damage from lack of pressure."
```

Comment:
Aren't you changing this last bit so they don't take damage from lack of pressure/I kinda get the pressure thing, but don't all crewmembers spawn with air?

Reply:
for the same reason the ash heretic gets nobreath, cold immunity, lowpressure immunity, high pressure immunity, hot immunity, as well as damage resistances. currently void heretic has zero perks/traits (guy who coded it said he forgot). so it may not be as good as the other paths perks, but its still something, and i mainly noticed it as an issue when the heretic spawns the void storm but wears a mask that cant use internals (the masks the heretics can get cant use internals) ... the thing with void storms is the freeze the air, lowering the o2 pressure, so if you dont have internals you will suffocate.... inside your very own storm. so its just one extra bonus that fits thematically with with the ascension path of void (why would a void aristrocrat need to breathe when he rules the void, aka an airless space?)

if anything the other paths need some traits removed, ash path for example.... why exactly is a fire heretic immune to cold? it doesnt really fit thematically but im focusing on the void path with this

Reply:
Do they still take damage from low pressure? because it looks like you gave them resistance to it.

Reply:
they do not, the finisher makes them immune from low pressure, seeing as they are the Aristocrat of the Void, it has been tested, it is full immunity, not a resistance. I have spoken with the guy who made heretics, he intended for Void Heretics to have immunities and stuff, he just forgot. even with just these two changes, Void heretic has not even 30% of the immunities that the other heretics get as their finisher, so i dont feel this to be particularly unbalanced, relatively speaking

https://user-images.githubusercontent.com/40489693/102828346-83d5d200-43b2-11eb-99a5-a92d43faf046.mp4

Reply:
You should update the text properly then, because it still talks about taking damage from low pressure

Reply:
no no, you misunderstand, the text your reviewing here is for the nobreathe, this is on the arisocrats way tech which is one of the first techs on the void path. you DO still take damage from low pressure at that point, full low pressure immunity only comes when you ascend.